modules/nf-core/kmcp/compute/main.nf

```diff
@@ -0,0 +1,37 @@
+process KMCP_COMPUTE {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "bioconda::kmcp=0.9.1"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/kmcp:0.9.1--h9ee0642_0':
+        'quay.io/biocontainers/kmcp:0.9.1--h9ee0642_0' }"
+
+    input:
+    tuple val(meta), path(sequences)
+
+    output:
+    tuple val(meta), path("tmp/*") , emit: tmp
+    path "versions.yml"            , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def input = sequences.isDirectory()? "--in-dir ${sequences}" : "${sequences}"
+    """
+    kmcp \\
+        compute \\
+        $args \\
+        --threads $task.cpus \\
+        --out-dir 'tmp/' \\
```

Comment:
Why `tmp/` and not `${prefix}/`?

Reply:
For some reason thought that tmp was standard name for the output directory.

Reply:
Fair enough - I guess change as above :)